micro-rdk/src/common/conn/viam.rs

```diff
@@ -1307,6 +1447,9 @@ mod tests {
                 "/viam.app.v1.RobotService/Log" => self.log(body.split_off(5)),
                 "/viam.app.v1.RobotService/NeedsRestart" => self.needs_restart(body.split_off(5)),
                 "/viam.app.v1.RobotService/Config" => self.get_config(),
+                "/viam.app.agent.v1.AgentDeviceService/DeviceAgentConfig" => {
+                    return Err(ServerError::new(GrpcError::RpcInternal, None));
```

Comment:
return unimplemented by default?

Reply:
Just to clarify, do you mean that this shouldn't panic, or that I should just change the error for this endpoint to unimplemented?